aws-cpp-sdk-core/source/client/CoreErrors.cpp

```diff
@@ -164,6 +166,10 @@ AWSError<CoreErrors> CoreErrorsMapper::GetErrorForName(const char* errorName)
   {
     return AWSError<CoreErrors>(CoreErrors::REQUEST_TIME_TOO_SKEWED, true);
   }
+  else if (errorHash == REQUEST_TIMEOUT_HASH)
+  {
+    return AWSError<CoreErrors>(CoreErrors::REQUEST_TIMEOUT, true);
```

Comment:
One thing is missing. Update `CoreErrorsMapper::GetErrorForHttpResponseCode` to map `HttpResponseCode::REQUEST_TIMEOUT` to `CoreErrors::REQUEST_TIMEOUT`

Reply:
Done in f37e1fff272baf4a9669ed6d593f58fb50a81ad7.